src/commands/wrap.ts

```diff
@@ -46,16 +43,45 @@ export const builder = (yargs: Argv): Argv =>
       description: "Use if raw directory contains wrapped files",
       type: "boolean",
       default: false
+    })
+    .option("output-file", {
+      alias: "of",
+      type: "string",
+      description: "Use if output path is a file",
+      conflicts: "output-dir"
+    })
+    .option("output-dir", {
+      alias: "od",
+      type: "string",
+      description: "Use if output path is a directory",
+      conflicts: "output-file"
     });
 
 export const handler = async (args: WrapCommand): Promise<string> => {
   try {
-    const merkleRoot = await wrap(args.rawDocumentsPath, args.wrappedDocumentsDir, {
+    const outputPathType = args.outputDir ? "directory" : args.outputFile ? "file" : "stdOut";
+    const outputPath =
```

Comment:
outputPath = args.outputDir || args.outputFile

if undefined => that means we need to use std out